### ISSUE TITLE
Model definition suddenly becomes invalid 1.5.1

### ISSUE THREAD

**Issue by Jedsters** [FrSkyRC/ETHOS-Feedback-Community#3579]
I have been trying to setup a new model template from scratch and started with the official 1.5.0 release. 

I made a lot of progress and fortunately have a a backup from around 24hours ago which loads correctly

[v tail 1.5.zip](https://github.com/FrSkyRC/ETHOS-Feedback-Community/files/14498639/v.tail.1.5.zip)

I then noticed that there was an new release of 1.5.1 which I upgraded to yesterday.

Today I was checking my model definitions and noticed a few things weren't working as desired so I've been correcting them. 
Please don't take this as a complete list, but in case it helps and as far as I can recall...

- The Ail=>Rud mixer had the Negative Option applied to the wrong Weights so both have been changed  
- I've changed Extra Trim1 from Easy Mode to Custom mode and added a number of Behaviours

I was then playing around with Trims and noticed an issue which I felt needed raising. To do so I switched to another basic model to take a couple of photos. However when I tried to switch back to my template I now just get Invalid Data

This is the bad model

[v tail 1.5.1.zip](https://github.com/FrSkyRC/ETHOS-Feedback-Community/files/14498640/v.tail.1.5.1.zip)

I'm just very glad that I had a backup from yesterday as it's taken many hours to create this template from scratch so far.

**Comment by Jedsters** [comment]
Update:
I've just recreated the problem!

I took the v tail 1.5.bin above, repeated the Ail=>Rud change, switched to a different model and back and that worked without problem.
I then changed Extra Trim1 to Custom mode and added the following:

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/96293240/9d54b3f1-481e-403b-b3f4-bc6afb6fc8fb)

again I switched to another model and back but this time I got invalid data.

Something in the above Behaviours therefore appears to result in bad data.
